lib/lib.go

```diff
@@ -291,7 +287,7 @@ func NewInstance(ctx context.Context, repoPath string, opts ...Option) (qri *Ins
 		opts = []Option{
 			OptStdIOStreams(),
 			OptSetIPFSPath(""),
-			OptCheckConfigMigrations(),
+			OptCheckConfigMigrations(false),
```

Comment:
What are your thoughts on changing the migration UI such that after migrations run successfully the qri binary exits with a message like "migration successful, please run command again". As it is now, if I run `qri list` then I get the migration prompt, confirm, see some messages about progress, then soon after the pager pops up and obscures what happened. I think it would be fine to bail out inside and ask the user to start over, it's less cognitive load than having to remember what was going on before the migration started.

Reply:
+1

Reply:
> "migration successful, please run command again"
fully agree

Reply:
nm, managed to fit this in

Reply:
Yes totally